### PR TITLE
jo: add new package

### PR DESCRIPTION
**Pull request by 1480c1**

### CODE REVIEW

File: jo/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+build() {
+  rm -rf "${srcdir}/build-${CARCH}"
+  mkdir -p "${srcdir}/build-${CARCH}"
+  cd "${srcdir}/build-${CARCH}"  || return 1
+
+  ../${pkgname}-${pkgver}/configure \
+    --prefix=/usr \
+    --build="${CARCH}" \
+    --host="${CARCH}"
```
Comment (lazka) [inline jo/PKGBUILD:21]: `${CHOST}`?

Comment (1480c1) [reply jo/PKGBUILD]: Accidentally replaced that with the wrong variable

**Comment by lazka** [comment]
Why do you need a msys version of this as well?

**Comment by 1480c1** [comment]
Was mainly doing this for the bash-completion, but I do notice that jq doesn't have a msys2 version, so probably actually unnecessary

**Comment by lazka** [comment]
I've created #2438 for bash-completion

**Pull request closed without merging**